lxa_iobus/node/remote_node.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from aiohttp import ClientResponseError, ClientSession
+
+from lxa_iobus.canopen import SdoAbort
+
+from .base_node import LxaBaseNode
+
+logger = logging.getLogger("lxa_iobus.remote_node")
+
+
+class LxaRemoteNode(LxaBaseNode):
+    @classmethod
+    async def new(cls, base_url, node_name):
```

Comment:
This is a new external interface. I would prefer if we would add some comments on how to use this.

Reply:
Hi, I've added a few docstrings to the `LxaRemoteNode` class now.
I've also looked into `LxaBaseNode`, but I think most of the methods there are quite self descriptive already (e.g. `ping`, `set_locator_state`, `invoke_isp`, `info`).

Reply:
Thanks for the update. For me this is now descriptive enough.